doc/cookbook/source/examples/clustering/gmm.rst

```diff
@@ -0,0 +1,49 @@
+=======================
+Gaussian Mixture Models
+=======================
+
+A Gaussian Mixture Model is a probabilistic model that assumes all the data points are generated from a mixture of a finite number of Gaussian distributions with unknown parameters.
```

Comment:
"that assumes that data are generated from a finite mixutre of Gaussians with unknown parameters" is better